pcsx2/gui/MemoryCardFile.cpp

```diff
@@ -359,7 +363,23 @@ s32 FileMemoryCard::Save( uint slot, const u8 *src, u32 adr, int size )
 	}
 
 	if( !Seek(mcfp, adr) ) return 0;
-	return mcfp.Write( m_currentdata.GetPtr(), size ) != 0;
+
+	int status = mcfp.Write( m_currentdata.GetPtr(), size );
+
+	if( 0 != status ) {
```

Comment:
Maybe ``if(status)`` instead? having a constant before the variable on relational operators doesn't really look nice. [**We had similar discussions before**](https://github.com/PCSX2/pcsx2/commit/36dd50005ad38de13a692ac9b26e09b7a3313147#commitcomment-18002868).